policy/cnspec_policy.proto

```diff
@@ -114,6 +116,8 @@ message Policy {
   string graph_content_checksum = 38;
   string local_execution_checksum = 39;
   string graph_execution_checksum = 40;
+  // timestamp when the policy was last applied
+  int64 last_applied = 48;
```

Comment:
I don't think this one makes sense. I would only expect this information in `PolicyRef`. What's the thought for how this one is used?